ecs-logs-collector.sh

```diff
@@ -419,19 +418,20 @@ get_docker_info()
 {
   try "gather Docker daemon information"
 
-  pgrep docker > /dev/null
+  ps -ef |grep dockerd |grep -v grep  >> /dev/null
   if [[ "$?" -eq 0 ]]; then
     mkdir -p ${info_system}/docker
 
-    docker info > ${info_system}/docker/docker-info.txt 2>&1
-    docker ps --all --no-trunc > ${info_system}/docker/docker-ps.txt 2>&1
-    docker images > ${info_system}/docker/docker-images.txt 2>&1
-    docker version > ${info_system}/docker/docker-version.txt 2>&1
+    timeout 75 docker info > ${info_system}/docker/docker-info.txt 2>&1 || echo "Timed out, ignoring \"docker info output \" "
+    timeout 75 docker ps --all --no-trunc > ${info_system}/docker/docker-ps.txt 2>&1 || echo "Timed out, ignoring \"docker ps --all --no-truc output \" "
+    timeout 75 docker images > ${info_system}/docker/docker-images.txt 2>&1 || echo "Timed out, ignoring \"docker images output \" "
+    timeout 75 docker version > ${info_system}/docker/docker-version.txt 2>&1 || echo "Timed out, ignoring \"docker version output \" "
 
     ok
 
   else
-    die "The Docker daemon is not running."
+    echo "WARNING: The Docker daemon is not running." > ${info_system}/docker/docker-not-running.txt
```

Comment:
This could be simplified to print out to the file and stdout:

```bash
echo "WARNING: The Docker daemon is not running." | tee ${info_system}/docker/docker-not-running.txt
```